

<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
  <meta charset="utf-8" />
  
    <meta property="article:modified_time" content="2023-06-08T08:33:47-04:00" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  
  <title>CellBaum User’s Guide &mdash; Galloway Lab Protocols  documentation</title>
  

  
  <link rel="stylesheet" href="../../_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/pygments.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/pygments.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/plot_directive.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/css/custom.css" type="text/css" />

  
  

  
  

  

  
  <!--[if lt IE 9]>
    <script src="../../_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
    
      <script type="text/javascript" id="documentation_options" data-url_root="../../" src="../../_static/documentation_options.js"></script>
        <script data-url_root="../../" id="documentation_options" src="../../_static/documentation_options.js"></script>
        <script src="../../_static/jquery.js"></script>
        <script src="../../_static/underscore.js"></script>
        <script src="../../_static/_sphinx_javascript_frameworks_compat.js"></script>
        <script src="../../_static/doctools.js"></script>
    
    <script type="text/javascript" src="../../_static/js/theme.js"></script>

    
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="next" title="CFSE Labeling" href="cfse_labeling.html" />
    <link rel="prev" title="ATAC-see" href="atac_see.html" /> 
</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">
    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search"  style="background: #2c6854" >
          

          
            <a href="../../index.html" class="icon icon-home"> Galloway Lab Protocols
          

          
          </a>

          
            
            
              <div class="version">
                ehs_safety
              </div>
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        
        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <ul class="current">
<li class="toctree-l1"><a class="reference internal" href="../../general/index.html">General Lab Maintenance</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../../general/centrifuge_motor_mount_replacement.html">Replacing microcentrifuge motor mounts</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../general/centrifuge_motor_mount_replacement.html#required-materials">Required materials</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../general/centrifuge_motor_mount_replacement.html#procedure">Procedure</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../general/clean_autoclave_guideline.html">Cleaning and autoclaving guidelines</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../general/clean_autoclave_guideline.html#washing-glassware-and-tools">Washing glassware and tools</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../general/clean_autoclave_guideline.html#glass-bottles-and-flasks">Glass bottles and flasks</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../general/clean_autoclave_guideline.html#glass-beads">Glass beads</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../general/clean_autoclave_guideline.html#dissection-tools">Dissection Tools</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../general/clean_autoclave_guideline.html#autoclaving">Autoclaving</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../general/clean_autoclave_guideline.html#tip-boxes">Tip boxes</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../general/clean_autoclave_guideline.html#tube-containers">Tube containers</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../general/clean_autoclave_guideline.html#glass-pipette-containers">Glass pipette containers</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../general/clean_autoclave_guideline.html#toothpicks">Toothpicks</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../general/clean_autoclave_guideline.html#flasks">Flasks</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../general/clean_autoclave_guideline.html#id1">Dissection tools</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../general/clean_autoclave_guideline.html#glass-bottles-containing-liquid-or-glass-beads">Glass bottles (containing liquid or glass beads)</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../general/clean_bsc.html">Cleaning BSCs</a></li>
<li class="toctree-l2"><a class="reference internal" href="../../general/cleaning_microcentrifuges.html">Cleaning microcentrifuges</a></li>
<li class="toctree-l2"><a class="reference internal" href="../../general/data_backup_check.html">Checking data backups</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../general/data_backup_check.html#onedrive-backups">Onedrive backups</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../general/data_backup_check.html#once-a-year-checks">Once a year checks</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../general/data_backup_check.html#ngs-backups">NGS backups</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../general/data_backup_check.html#adding-a-new-ngs-dataset-checklist">Adding a new NGS dataset checklist</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../general/data_backup_check.html#once-a-year-checklist">Once a year checklist</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../general/incubator_decon.html">Incubator decontamination</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../general/incubator_decon.html#maui-and-moana-main-tc">Maui and Moana (Main TC)</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../general/incubator_decon.html#hei-hei-and-pua-main-tc">Hei Hei and Pua (Main TC)</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../general/incubator_decon.html#te-ka-quarantine-and-te-fiti-66-219">Te Kā (Quarantine) and Te Fiti (66-219)</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../general/onedrive_user_change.html">Replacing the logged-in OneDrive user</a></li>
<li class="toctree-l2"><a class="reference internal" href="../../general/ordering.html">Ordering</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../general/ordering.html#direct-ordering-through-coupa">Direct ordering through Coupa</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../general/ordering.html#requests-through-quartzy">Requests through Quartzy</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../general/ordering.html#bulk-plastics-order">Bulk Plastics Order</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../general/recycling.html">Recycling</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../../instruments/index.html">Instrument Use and Core Facility Access</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../../instruments/attune/index.html">Attune operation</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../instruments/attune/startup_shutdown.html">Attune startup and shutdown</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../instruments/attune/startup_shutdown.html#quick-startup-checklist">Quick startup checklist</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../instruments/attune/startup_shutdown.html#quick-shutdown-checklist">Quick shutdown checklist</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../instruments/attune/startup_shutdown.html#initial-inspection-and-startup">Initial inspection and startup</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../instruments/attune/startup_shutdown.html#performance-test">Performance test</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../instruments/attune/startup_shutdown.html#shutdown">Shutdown</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../instruments/attune/attune_operation.html">Attune operation</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../instruments/attune/attune_operation.html#procedure-descriptions">Procedure descriptions</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../instruments/attune/attune_operation.html#weekly-system-cleaning">Weekly system cleaning</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../instruments/attune/attune_operation.html#every-3-6-month-system-decontamination">Every ~3-6 month system decontamination</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../instruments/attune/attune_operation.html#focusing-fluid-refill">Focusing fluid refill</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../instruments/attune/attune_operation.html#full-attune-guides">Full Attune guides</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../instruments/attune/user_replacements.html">User-replaceable parts</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../instruments/attune/user_replacements.html#main-syringe">Main syringe</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../instruments/attune/user_replacements.html#sample-injection-probe-sip">Sample Injection Probe (SIP)</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../instruments/attune/user_replacements.html#focusing-fluid-filters">Focusing fluid filters</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../instruments/attune/empirical_troubleshooting.html">Attune troubleshooting</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../instruments/attune/empirical_troubleshooting.html#delay-before-events-start-running">Delay before events start running</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../instruments/attune/empirical_troubleshooting.html#fluid-leak-from-the-autosampler-sip-onto-a-running-plate">Fluid leak from the autosampler SIP onto a running plate</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../instruments/attune/empirical_troubleshooting.html#wells-skipping-and-the-autosampler-fails-to-draw-focusing-fluid">Wells skipping and the autosampler fails to draw focusing fluid</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../instruments/attune/empirical_troubleshooting.html#power-cycling-the-attune">Power cycling the Attune</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../instruments/core_facilities/Koch_flow.html">Koch Flow Core</a></li>
<li class="toctree-l2"><a class="reference internal" href="../../instruments/core_facilities/animal_facility.html">Animal Facility</a></li>
<li class="toctree-l2"><a class="reference internal" href="../../instruments/core_facilities/keck.html">Keck Imaging Facility</a></li>
<li class="toctree-l2"><a class="reference internal" href="../../instruments/core_facilities/myco_testing.html">Myco Testing</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../instruments/core_facilities/myco_testing.html#getting-access">Getting Access</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../instruments/core_facilities/myco_testing.html#submitting-testing-request-in-ilab">Submitting Testing Request in iLab</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../instruments/core_facilities/myco_testing.html#submitting-samples-for-testing">Submitting Samples for Testing</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../instruments/core_facilities/myco_testing.html#results">Results</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../instruments/core_facilities/supercloud.html">MIT SuperCloud Access</a></li>
<li class="toctree-l2"><a class="reference internal" href="../../instruments/keyence/keyence_use.html">Keyence beginner’s user guide</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../instruments/keyence/keyence_use.html#basic-operation">Basic Operation</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../instruments/keyence/keyence_use.html#multi-point-capture">Multi-point capture</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../instruments/keyence/keyence_use.html#z-stack">Z-stack</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../instruments/keyence/keyence_use.html#stitching">Stitching</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../instruments/keyence/timelapse.html">Keyence Time Lapse Imaging</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../instruments/keyence/timelapse.html#setting-up-the-incubation-chamber">Setting up the incubation chamber</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../instruments/keyence/timelapse.html#using-the-keyence-software">Using the Keyence Software</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../instruments/keyence/timelapse.html#tips-and-tricks">Tips and Tricks</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l1 current"><a class="reference internal" href="../index.html">Protocols</a><ul class="current">
<li class="toctree-l2 current"><a class="reference internal" href="index.html">Biochemical and analytical protocols</a><ul class="current">
<li class="toctree-l3"><a class="reference internal" href="EU.html">Homemade EU click</a><ul>
<li class="toctree-l4"><a class="reference internal" href="EU.html#stock-solutions">Stock solutions</a></li>
<li class="toctree-l4"><a class="reference internal" href="EU.html#add-eu-to-cells">Add EU to cells</a></li>
<li class="toctree-l4"><a class="reference internal" href="EU.html#dissociate">Dissociate</a></li>
<li class="toctree-l4"><a class="reference internal" href="EU.html#fix-cells">Fix cells</a></li>
<li class="toctree-l4"><a class="reference internal" href="EU.html#permeabilize">Permeabilize</a></li>
<li class="toctree-l4"><a class="reference internal" href="EU.html#prepare-click-label-mix">Prepare click/label mix</a></li>
<li class="toctree-l4"><a class="reference internal" href="EU.html#label-and-wash">Label and wash</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="HIVE.html">HIVE scRNA-seq</a><ul>
<li class="toctree-l4"><a class="reference internal" href="HIVE.html#sample-capture">Sample capture</a></li>
<li class="toctree-l4"><a class="reference internal" href="HIVE.html#hive-processing">HIVE processing</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="antibody_staining.html">Immunofluorescent Staining</a><ul>
<li class="toctree-l4"><a class="reference internal" href="antibody_staining.html#solutions-required">Solutions Required</a></li>
<li class="toctree-l4"><a class="reference internal" href="antibody_staining.html#adherent-cell-staining">Adherent Cell Staining</a></li>
<li class="toctree-l4"><a class="reference internal" href="antibody_staining.html#cell-staining-for-flow">Cell Staining for Flow</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="atac_see.html">ATAC-see</a><ul>
<li class="toctree-l4"><a class="reference internal" href="atac_see.html#transposase-buffer-preparation">Transposase buffer preparation</a></li>
<li class="toctree-l4"><a class="reference internal" href="atac_see.html#id1">ATAC-see</a></li>
</ul>
</li>
<li class="toctree-l3 current"><a class="current reference internal" href="#">CellBaum User’s Guide</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#local-usage">Local Usage</a></li>
<li class="toctree-l4"><a class="reference internal" href="#mit-supercloud-usage">MIT Supercloud Usage</a></li>
<li class="toctree-l4"><a class="reference internal" href="#data-analysis">Data Analysis</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="cfse_labeling.html">CFSE Labeling</a><ul>
<li class="toctree-l4"><a class="reference internal" href="cfse_labeling.html#celltrace-dyes-available-in-lab">CellTrace dyes available in lab</a></li>
<li class="toctree-l4"><a class="reference internal" href="cfse_labeling.html#other-materials-needed">Other materials needed:</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="dna_fish.html">HCR DNA-FISH</a><ul>
<li class="toctree-l4"><a class="reference internal" href="dna_fish.html#fixation">Fixation</a></li>
<li class="toctree-l4"><a class="reference internal" href="dna_fish.html#hybridization">Hybridization</a></li>
<li class="toctree-l4"><a class="reference internal" href="dna_fish.html#probe-preperation">Probe Preperation</a></li>
<li class="toctree-l4"><a class="reference internal" href="dna_fish.html#post-hybridization">Post-Hybridization</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="halo_labeling.html">Halo-tag Labeling</a><ul>
<li class="toctree-l4"><a class="reference internal" href="halo_labeling.html#halo-tag-cell-permeable-dyes-available-in-lab">Halo-tag cell permeable dyes available in lab</a></li>
<li class="toctree-l4"><a class="reference internal" href="halo_labeling.html#other-materials-needed">Other materials needed:</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="hcr.html">HCR RNA-FISH</a><ul>
<li class="toctree-l4"><a class="reference internal" href="hcr.html#solution-detection-stage">Solution Detection Stage</a></li>
<li class="toctree-l4"><a class="reference internal" href="hcr.html#solution-amplification-stage">Solution Amplification Stage</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="hoechst_staining.html">Live nuclear Staining using Hoechst</a><ul>
<li class="toctree-l4"><a class="reference internal" href="hoechst_staining.html#solutions-required">Solutions Required</a></li>
<li class="toctree-l4"><a class="reference internal" href="hoechst_staining.html#live-hoechst-staining">Live Hoechst staining</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="image_segmentation.html">Image segmentation and particle analysis in ImageJ</a><ul>
<li class="toctree-l4"><a class="reference internal" href="image_segmentation.html#defining-regions-of-interest-roi">Defining regions of interest (ROI)</a></li>
<li class="toctree-l4"><a class="reference internal" href="image_segmentation.html#quantifying-nuclear-localized-foci">Quantifying nuclear-localized foci</a></li>
<li class="toctree-l4"><a class="reference internal" href="image_segmentation.html#merging-images-in-imagej">Merging images in ImageJ</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="kapa_sybr_fast_qpcr.html">KAPA SYBR® FAST qPCR</a><ul>
<li class="toctree-l4"><a class="reference internal" href="kapa_sybr_fast_qpcr.html#protocol">Protocol</a></li>
<li class="toctree-l4"><a class="reference internal" href="kapa_sybr_fast_qpcr.html#master-mix-preparation">1. Master Mix Preparation</a></li>
<li class="toctree-l4"><a class="reference internal" href="kapa_sybr_fast_qpcr.html#reaction-setup">2. Reaction Setup</a></li>
<li class="toctree-l4"><a class="reference internal" href="kapa_sybr_fast_qpcr.html#qpcr">3. qPCR</a></li>
<li class="toctree-l4"><a class="reference internal" href="kapa_sybr_fast_qpcr.html#data-analysis">4. Data Analysis</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="magnetic_bead.html">Magnetic bead clean-up for genomics libraries</a><ul>
<li class="toctree-l4"><a class="reference internal" href="magnetic_bead.html#uses">Uses</a></li>
<li class="toctree-l4"><a class="reference internal" href="magnetic_bead.html#determine-bead-volume-to-use-by-size-range-you-wish-to-recover">Determine Bead Volume to use by size range you wish to recover:</a></li>
<li class="toctree-l4"><a class="reference internal" href="magnetic_bead.html#preparations">Preparations:</a></li>
<li class="toctree-l4"><a class="reference internal" href="magnetic_bead.html#basic-bead-clean-protocol">Basic Bead Clean Protocol:</a></li>
<li class="toctree-l4"><a class="reference internal" href="magnetic_bead.html#step-big-bead-clean-to-remove-large-fragments-350bp">2-step “Big” bead clean to remove large fragments &gt;~350bp.</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="nuclei_isolation_genomics.html">Nuclei Isolation for Downstream Genomics Applications</a><ul>
<li class="toctree-l4"><a class="reference internal" href="nuclei_isolation_genomics.html#nuclear-isolation">Nuclear Isolation</a></li>
<li class="toctree-l4"><a class="reference internal" href="nuclei_isolation_genomics.html#recipes">Recipes</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="psoralen_qpcr.html">Psoralen-qPCR supercoiling assay</a><ul>
<li class="toctree-l4"><a class="reference internal" href="psoralen_qpcr.html#experiment-setup">Experiment setup</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="sci_ATAC_seq.html">sci-ATAC-seq</a></li>
<li class="toctree-l3"><a class="reference internal" href="single_cell_qpcr.html">Single cell qPCR</a><ul>
<li class="toctree-l4"><a class="reference internal" href="single_cell_qpcr.html#single-cell-acquisition">Single-cell acquisition</a></li>
<li class="toctree-l4"><a class="reference internal" href="single_cell_qpcr.html#lysis-reverse-transcription-rt-and-specific-target-amplification-sta">Lysis, reverse transcription (RT), and specific target amplification (STA)</a></li>
<li class="toctree-l4"><a class="reference internal" href="single_cell_qpcr.html#single-cell-qpcr-on-fluidigm-biomark">Single-cell qPCR on Fluidigm Biomark</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="single_shot_rna_isolation_and_rt.html">SingleShot RNA isolation and Reverse Transcription</a><ul>
<li class="toctree-l4"><a class="reference internal" href="single_shot_rna_isolation_and_rt.html#materials-need-for-rna-isolation">Materials need for RNA isolation</a></li>
<li class="toctree-l4"><a class="reference internal" href="single_shot_rna_isolation_and_rt.html#materials-needed-for-cdna-synthesis">Materials needed for cDNA synthesis</a></li>
<li class="toctree-l4"><a class="reference internal" href="single_shot_rna_isolation_and_rt.html#rna-isolation-using-singleshot-cell-lysis">RNA isolation using SingleShot Cell Lysis</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="snap_labeling.html">Snap-tag Labeling</a><ul>
<li class="toctree-l4"><a class="reference internal" href="snap_labeling.html#snap-tag-cell-permeable-dyes-available-in-lab">Snap-tag cell permeable dyes available in lab</a></li>
<li class="toctree-l4"><a class="reference internal" href="snap_labeling.html#other-materials-needed">Other materials needed:</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="tn5_titration.html">Titration of Tn5 transposition in MEFs</a><ul>
<li class="toctree-l4"><a class="reference internal" href="tn5_titration.html#part-1-assembling-the-transposase">Part 1. Assembling the Transposase</a></li>
<li class="toctree-l4"><a class="reference internal" href="tn5_titration.html#part-1-1-prepping-the-tn5-concentrations-from-assembled-tn5-transposome-per-transposition-reaction-50ul">Part 1.1 Prepping the Tn5 concentrations from Assembled Tn5 transposome per Transposition reaction (50uL)</a></li>
<li class="toctree-l4"><a class="reference internal" href="tn5_titration.html#part-2-transposition-reaction-of-isolated-nuclei">Part 2. Transposition Reaction of Isolated nuclei</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="western_blot.html">Western Blot</a><ul>
<li class="toctree-l4"><a class="reference internal" href="western_blot.html#lysis">Lysis</a></li>
<li class="toctree-l4"><a class="reference internal" href="western_blot.html#bradford-assay">Bradford Assay</a></li>
<li class="toctree-l4"><a class="reference internal" href="western_blot.html#protein-gel-casting">Protein Gel Casting</a></li>
<li class="toctree-l4"><a class="reference internal" href="western_blot.html#loading-and-running-the-gel">Loading and Running the Gel</a></li>
<li class="toctree-l4"><a class="reference internal" href="western_blot.html#coomassie-staining">Coomassie Staining</a></li>
<li class="toctree-l4"><a class="reference internal" href="western_blot.html#transferring-the-protein-from-the-gel-to-the-membrane">Transferring the protein from the gel to the membrane</a></li>
<li class="toctree-l4"><a class="reference internal" href="western_blot.html#antibody-staining">Antibody Staining</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../cloning/index.html">Cloning protocols</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../cloning/gibson.html">Gibson assembly</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../cloning/gibson.html#protocol">Protocol</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../cloning/glycerol_stocking.html">Glycerol Stocking</a></li>
<li class="toctree-l3"><a class="reference internal" href="../cloning/golden_gate.html">Golden Gate assembly</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../cloning/golden_gate.html#protocol-for-pcr-fragments">Protocol for PCR fragments</a></li>
<li class="toctree-l4"><a class="reference internal" href="../cloning/golden_gate.html#protocol-for-ppv-plasmids">Protocol for pPV plasmids</a></li>
<li class="toctree-l4"><a class="reference internal" href="../cloning/golden_gate.html#reference-for-ppv-connector-sequences">Reference for pPV connector sequences</a></li>
<li class="toctree-l4"><a class="reference internal" href="../cloning/golden_gate.html#reference-for-entire-golden-gate-workflow">Reference for entire Golden Gate workflow</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../cloning/ligation_assembly.html">Ligation assembly</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../cloning/ligation_assembly.html#protocol">Protocol</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../cloning/lr_cloning.html">LR cloning</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../cloning/lr_cloning.html#protocol">Protocol</a></li>
<li class="toctree-l4"><a class="reference internal" href="../cloning/lr_cloning.html#expected-results">Expected results</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../cloning/oligo_annealing.html">Oligo annealing for ligation cloning</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../cloning/oligo_annealing.html#oligo-phosphorylation">Oligo phosphorylation</a></li>
<li class="toctree-l4"><a class="reference internal" href="../cloning/oligo_annealing.html#oligo-annealing">Oligo annealing</a></li>
<li class="toctree-l4"><a class="reference internal" href="../cloning/oligo_annealing.html#combined-oligo-phosphorylation-and-annealing">Combined oligo phosphorylation and annealing</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../cloning/touchdown_PCR.html">Touchdown PCR</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../cloning/touchdown_PCR.html#id1">Touchdown PCR</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../cloning/transformation.html">Bacterial Transformation</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../cloning/transformation.html#cloning-summary">Cloning Summary</a></li>
<li class="toctree-l4"><a class="reference internal" href="../cloning/transformation.html#transformation-chemically-competent-bacteria">Transformation (Chemically Competent Bacteria)</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../in_vitro_transcription/index.html">in vitro transcription</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../in_vitro_transcription/DNA_template_design.html">Design and preparation of DNA IVT template</a></li>
<li class="toctree-l3"><a class="reference internal" href="../in_vitro_transcription/DNA_template_design.html#creating-a-new-ivt-template-plasmid">Creating a new IVT template plasmid</a></li>
<li class="toctree-l3"><a class="reference internal" href="../in_vitro_transcription/DNA_template_design.html#generating-ivt-linear-dna-template-from-an-ivt-template-plasmid">Generating IVT linear DNA template from an IVT template plasmid</a></li>
<li class="toctree-l3"><a class="reference internal" href="../in_vitro_transcription/mRNA_synthesis.html">modRNA synthesis</a></li>
<li class="toctree-l3"><a class="reference internal" href="../in_vitro_transcription/mRNA_synthesis.html#ivt-reaction">IVT reaction</a></li>
<li class="toctree-l3"><a class="reference internal" href="../in_vitro_transcription/mRNA_transfection.html">mRNA transfection</a></li>
<li class="toctree-l3"><a class="reference internal" href="../in_vitro_transcription/mRNA_transfection.html#messengermax-transfection-protocol">MessengerMax transfection protocol</a></li>
<li class="toctree-l3"><a class="reference internal" href="../in_vitro_transcription/mRNA_transfection.html#stemmacs-transfection-protocol">StemMACS transfection protocol</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../protein_production/index.html">Protein production</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../protein_production/bis_tris_protein_gels.html">Protein gel casting</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../protein_production/bis_tris_protein_gels.html#required-stock-solutions">Required stock solutions</a></li>
<li class="toctree-l4"><a class="reference internal" href="../protein_production/bis_tris_protein_gels.html#casting-protocol">Casting protocol</a></li>
<li class="toctree-l4"><a class="reference internal" href="../protein_production/bis_tris_protein_gels.html#gel-casting-setup">Gel casting setup</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../protein_production/chitin_protein_purification.html">Chitin-binding protein purification</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../protein_production/chitin_protein_purification.html#required-solutions">Required solutions</a></li>
<li class="toctree-l4"><a class="reference internal" href="../protein_production/chitin_protein_purification.html#protocol">Protocol</a></li>
<li class="toctree-l4"><a class="reference internal" href="../protein_production/chitin_protein_purification.html#dialysis-and-concentration-protocol">Dialysis and concentration protocol</a></li>
<li class="toctree-l4"><a class="reference internal" href="../protein_production/chitin_protein_purification.html#regeneration-protocol">Regeneration Protocol</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../protein_production/denaturing_protein_gel.html">Denaturing protein gel run and staining</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../protein_production/denaturing_protein_gel.html#solutions-required">Solutions required</a></li>
<li class="toctree-l4"><a class="reference internal" href="../protein_production/denaturing_protein_gel.html#running-procedure">Running procedure</a></li>
<li class="toctree-l4"><a class="reference internal" href="../protein_production/denaturing_protein_gel.html#staining-coomassie">Staining (Coomassie)</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../protein_production/protein_expression.html">Protein expression</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../protein_production/protein_expression.html#protocol">Protocol</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../protein_production/protein_purification.html">His-tag protein purification</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../protein_production/protein_purification.html#source">Source</a></li>
<li class="toctree-l4"><a class="reference internal" href="../protein_production/protein_purification.html#required-solutions">Required solutions</a></li>
<li class="toctree-l4"><a class="reference internal" href="../protein_production/protein_purification.html#protocol">Protocol</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../safety/index.html">Safety</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../safety/virus_safety.html">Virus safety</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../safety/virus_safety.html#control-methods">Control methods</a></li>
<li class="toctree-l4"><a class="reference internal" href="../safety/virus_safety.html#virus-protocols">Virus protocols</a></li>
<li class="toctree-l4"><a class="reference internal" href="../safety/virus_safety.html#exposure-plan">Exposure plan</a></li>
<li class="toctree-l4"><a class="reference internal" href="../safety/virus_safety.html#lessons-learned-from-incidents">Lessons learned from incidents</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../tc/index.html">TC protocols</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../tc/tc-basics/index.html">TC Basics</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../tc/tc-basics/TC_basics.html">TC Best Practices</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/tc-basics/adherent_cell_culture.html">Adherent Cell Culture</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/tc-basics/cryopreservation_and_thawing.html">Mammalian cells cryopreservation and plating from frozen stock</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/tc-basics/transient_transfection.html">Transient HEK293T Transfection</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/tc-basics/flow_cytometry.html">Flow Cytometry</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/tc-basics/glass_slide.html">Glass slide cell culture</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/tc-basics/ipsc_culture.html">Culturing Human iPSCs</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/tc-basics/platE_MEF_reprogramming.html">Plat-E Retroviral Transduction of MEFs</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../tc/virus/index.html">Virus</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../tc/virus/virus_production.html">Virus production in HEK293T</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/virus/virus_infection.html">Transduction with concentrated virus</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/virus/platE_prod.html">Plat-E Retroviral Production</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/virus/VSV_production.html">VSV production</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/virus/moi.html">MOI Calculation</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../tc/SNAP_circuit.html">SNAP circuit (Transient expression in 96-well plate version)</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../tc/SNAP_circuit.html#materials">Materials</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/SNAP_circuit.html#procedure">Procedure</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../tc/cell_line_creation.html">Cell Line Creation</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../tc/cell_line_creation.html#day-1">Day -1</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/cell_line_creation.html#day-0">Day 0</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/cell_line_creation.html#id1">Day 1</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/cell_line_creation.html#day-2-4">Day 2-4</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/cell_line_creation.html#day-5-week-2">Day 5 - Week 2</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/cell_line_creation.html#week-2-1-day">Week 2 + 1 day</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/cell_line_creation.html#week-3">Week 3</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/cell_line_creation.html#clonal-selection-or-enrichment-via-flow-sorting">Clonal selection or enrichment via flow sorting</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/cell_line_creation.html#repicking">Repicking</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/cell_line_creation.html#genotyping-your-line">Genotyping your line</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../tc/electroporation_of_plasmids_into_MEFs.html">Electroporation of plasmids into MEFs</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../tc/electroporation_of_plasmids_into_MEFs.html#overexpression">Overexpression</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/electroporation_of_plasmids_into_MEFs.html#plasmid-concentration-experiment">Plasmid Concentration Experiment</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../tc/transgene_expression_via_ribozyme_switch.html">Transgene expression control through ribozyme switch (96-well plate version)</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../tc/transgene_expression_via_ribozyme_switch.html#materials">Materials</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/transgene_expression_via_ribozyme_switch.html#day-1-seed-cells">Day 1: Seed cells</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/transgene_expression_via_ribozyme_switch.html#day-2-transfection">Day 2: Transfection</a></li>
<li class="toctree-l4"><a class="reference internal" href="../tc/transgene_expression_via_ribozyme_switch.html#day-3-or-4">Day 3 or 4</a></li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../../recipes/index.html">Recipes</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../../recipes/bacteria/index.html">Bacterial recipes</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../recipes/bacteria/LBmedia.html">LB (Amp/Kan) plate</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../recipes/bacteria/antibiotic_stock.html">Antibiotic Stock Recipes</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/bacteria/antibiotic_stock.html#x-ampicillin-amp">1000X Ampicillin (AMP)</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/bacteria/antibiotic_stock.html#x-kanamycin-kan">1000X Kanamycin (KAN)</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/bacteria/antibiotic_stock.html#x-chloramphenicol-chlor">1000X Chloramphenicol (Chlor)</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../recipes/bacteria/competent_cells.html">Competent cells (NEB stable)</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/bacteria/competent_cells.html#day-1-culture-neb-stable-cells-for-overnight">Day 1. Culture NEB stable cells for overnight.</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/bacteria/competent_cells.html#day-2-grow-the-competent-cells-and-stock-the-cells">Day 2. Grow the competent cells and stock the cells.</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/bacteria/competent_cells.html#day-4-confirm-the-cell-growth">Day 4. Confirm the cell growth</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../recipes/bacteria/dna_wash.html">DNA wash buffer</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/bacteria/dna_wash.html#dna-wash-stock-solution-5x">DNA wash stock solution (5x)</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/bacteria/dna_wash.html#id1">DNA wash buffer</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../recipes/bacteria/media.html">Bacterial media recipes</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/bacteria/media.html#tb-terrific-broth">TB (Terrific Broth)</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/bacteria/media.html#x-tb-salts">10x TB salts</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/bacteria/media.html#soc-media">SOC Media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/bacteria/media.html#lb-media">LB Media</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../recipes/bacteria/othersolutions.html">Other Solutions</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/bacteria/othersolutions.html#glycerol-60">Glycerol (60%)</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/bacteria/othersolutions.html#xtae-buffer">1xTAE buffer</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../recipes/fish/index.html">FISH recipes</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../recipes/fish/fish_buffers.html">FISH buffer recipes</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/fish/fish_buffers.html#probe-hybridization-buffer">Probe Hybridization Buffer</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/fish/fish_buffers.html#probe-wash-buffer">Probe Wash Buffer</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/fish/fish_buffers.html#probe-amplification-buffer">Probe Amplification Buffer</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/fish/fish_buffers.html#x-ssct">5X SSCT</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../recipes/tc/index.html">TC recipes</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../recipes/tc/FBS_heatinact.html">FBS heat inactivation</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../recipes/tc/common_TC_aliquots.html">Common aliquot sizes for TC</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/common_TC_aliquots.html#x-neurotrophic-factor-stocks">1,000X Neurotrophic factor stocks</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/common_TC_aliquots.html#repsox">RepSox</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/common_TC_aliquots.html#fetal-bovine-serum-fbs-aliquots">Fetal Bovine Serum (FBS) Aliquots</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/common_TC_aliquots.html#x-pen-strep-aliquots">100X Pen-strep aliquots</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/common_TC_aliquots.html#x-geltrex-aliquots">50X Geltrex aliquots</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../recipes/tc/media.html">TC media recipes</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#n3-media">N3 media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#mef-media">MEF media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#breast-cancer-1-media">Breast cancer 1 media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#breast-cancer-2-media">Breast cancer 2 media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#breast-cancer-3-media">Breast cancer 3 media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#breast-cancer-4-media">Breast cancer 4 media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#epithelial-cell-1-media">Epithelial cell 1 media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#epithelial-cell-2-media">Epithelial cell 2 media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#glia-media">Glia media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#sorting-collection-media">Sorting/Collection media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#motor-neuron-dissociation-media">Motor neuron dissociation media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#freezing-media">Freezing media</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/media.html#hepes-buffered-dmem">HEPES-buffered DMEM</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../recipes/tc/other-tc-solutions.html">Other TC Solutions</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/other-tc-solutions.html#phosphate-buffered-saline-pbs">Phosphate buffered saline (PBS)</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/other-tc-solutions.html#gelatin">0.1% Gelatin</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../recipes/tc/pei.html">PEI Stocks</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/pei.html#materials">Materials</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/pei.html#dissolve-and-aliquot-pei">Dissolve and Aliquot PEI</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/pei.html#test-pei-ratio">Test PEI Ratio</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/pei.html#reference">Reference</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../recipes/tc/small_mol_stocks.html">Small molecule stocks for TC</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/small_mol_stocks.html#indole-3-acetic-acid-auxin-iaa">Indole-3-acetic acid (Auxin, IAA)</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/small_mol_stocks.html#indole-3-acetamide-auxin-precursor-iam">Indole-3-acetamide (Auxin precursor, IAM)</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/small_mol_stocks.html#mm-o6-benzylguanine-o6bg">50mM O6-Benzylguanine (O6BG)</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../recipes/tc/small_mol_stocks.html#mm-guanine">25mM Guanine</a></li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../../plot_gallery/index.html">Plot Gallery</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../../plot_gallery/exporting_high_quality_figures.html">High-quality PDF figure export</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../plot_gallery/exporting_high_quality_figures.html#intro">Intro</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../plot_gallery/exporting_high_quality_figures.html#problematic-features">Problematic features</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../plot_gallery/exporting_high_quality_figures.html#transparency-opacity">Transparency/opacity</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../plot_gallery/exporting_high_quality_figures.html#gradients-blurs">Gradients/blurs</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../plot_gallery/exporting_high_quality_figures.html#final-figure-export">Final figure export</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../plot_gallery/exporting_high_quality_figures.html#illustrator">Illustrator</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../plot_gallery/exporting_high_quality_figures.html#matplotlib-python">Matplotlib/Python</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../plot_gallery/exporting_high_quality_figures.html#preflight">Preflight</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../plot_gallery/flow_with_adjunct_hist.html">Flow plots with adjunct histograms</a></li>
<li class="toctree-l2"><a class="reference internal" href="../../plot_gallery/histograms.html">Histograms</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../plot_gallery/histograms.html#function-to-make-subplots-for-several-variables">Function to make subplots for several variables</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../plot_gallery/mixed_distribution_mean_plots.html">Mixed distribution + mean plots</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../plot_gallery/mixed_distribution_mean_plots.html#gating-plot">Gating plot</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../plot_gallery/mixed_distribution_mean_plots.html#box-plot-with-well-means">Box plot with well means</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../plot_gallery/mixed_distribution_mean_plots.html#violin-plot-with-well-means">Violin plot with well means</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../plot_gallery/multi_axes_plots.html">Multiple-axes plots</a></li>
<li class="toctree-l2"><a class="reference internal" href="../../plot_gallery/test.html">Testing matplotlib directives</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../plot_gallery/test.html#subheading">Subheading</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../plot_gallery/index.html#notes-on-reproducing-examples">Notes on reproducing examples</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../../writing_guide/index.html">Writing Guide</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../../writing_guide/Additional_resources.html">Additional resources</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../writing_guide/Additional_resources.html#big-picture-resources">Big picture resources</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../writing_guide/Additional_resources.html#getting-into-the-finer-details">Getting into the finer details</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../writing_guide/PointsOfStyle.html">Points of style</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../writing_guide/PointsOfStyle.html#specific-and-nonspecific-subjects">Specific and nonspecific subjects</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../writing_guide/PointsOfStyle.html#hyphenation">Hyphenation</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../writing_guide/PointsOfStyle.html#numbers-in-text">Numbers in text</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../writing_guide/PointsOfStyle.html#how-to-write-gene-and-protein-names">How to write gene and protein names?</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../writing_guide/PointsOfStyle.html#references">References</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../writing_guide/guiding_principles.html">Guiding principles</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../writing_guide/guiding_principles.html#tips">Tips</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../writing_guide/guiding_principles.html#outline">Outline</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../writing_guide/guiding_principles.html#active-voice">Active Voice</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../writing_guide/guiding_principles.html#editing-and-revising">Editing and revising</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../writing_guide/index.html#what-is-a-paper">What is a paper?</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../../bootcamp/index.html">Bootcamp</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../../bootcamp/iap/index.html">IAP Bootcamp</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../bootcamp/iap/data_analysis_with_python.html">Data analysis with Python</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/data_analysis_with_python.html#why-do-data-analysis-with-python">Why do data analysis with Python?</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/data_analysis_with_python.html#introduction-the-research-problem">Introduction: the research problem</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/data_analysis_with_python.html#good-file-layout-practices">Good file layout practices</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/data_analysis_with_python.html#oh-yaml-my-yaml">Oh yaml, my yaml</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/data_analysis_with_python.html#loading-flow-data">Loading flow data</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/data_analysis_with_python.html#introduction-to-pandas">Introduction to Pandas</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/data_analysis_with_python.html#plotting-with-seaborn">Plotting with Seaborn</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../bootcamp/iap/day_0_setup.html">Day 0: Software and training setup</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/day_0_setup.html#ehs-setup-and-trainings">EHS setup and trainings</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/day_0_setup.html#software-and-webservices">Software and webservices</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../bootcamp/iap/environment_check.html">Computational environment check</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/environment_check.html#initial-environment-check">Initial environment check</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/environment_check.html#protocols-check">Protocols check</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/environment_check.html#completion-date">Completion date</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../bootcamp/iap/git_intro.html">An introduction to Git</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/git_intro.html#acknowledgements">Acknowledgements</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/git_intro.html#motivation">Motivation</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/git_intro.html#git-by-example">Git by example</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/git_intro.html#basic-git-terms">Basic Git terms</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/git_intro.html#downloading-or-starting-a-repository">Downloading or starting a repository</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/git_intro.html#basic-history-terms-and-git-status-what-s-happening">Basic history terms and git status: What’s happening?</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/git_intro.html#branching-out-adding-and-checking-out-files">Branching out, adding, and checking out files</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/git_intro.html#snapshotting-your-work">Snapshotting your work</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/git_intro.html#ignoring-files">Ignoring files</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/git_intro.html#recovering-history-reverting-commits-and-file-level-checkouts">Recovering history: reverting commits and file-level checkouts</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/git_intro.html#working-with-others-remotes-and-inevitable-merge-conflicts">Working with others: remotes and inevitable merge conflicts</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/git_intro.html#conclusion">Conclusion</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../bootcamp/iap/software_tips_and_tricks.html">Software Tips and Tricks</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/software_tips_and_tricks.html#one-date-format-to-rule-them-all-naming">One date format to rule them all/naming</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/software_tips_and_tricks.html#paper-rss-feeds">Paper RSS feeds</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/software_tips_and_tricks.html#creating-feeds-from-searches">Creating feeds from searches</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/software_tips_and_tricks.html#zotero">Zotero</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/software_tips_and_tricks.html#better-quartzy">Better Quartzy</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/software_tips_and_tricks.html#regex-help">Regex help</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/software_tips_and_tricks.html#fonts">Fonts</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/software_tips_and_tricks.html#activate-snapgene-remotely">Activate SnapGene remotely</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../bootcamp/iap/startup_checklist.html">Startup checklist when working with repositories</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/startup_checklist.html#new-repository-python">New repository (Python)</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/startup_checklist.html#new-repository-julia">New repository (Julia)</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/startup_checklist.html#new-repository-r">New repository (R)</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/startup_checklist.html#existing-non-data-driven-repository-e-g-protocols">Existing, non-data-driven repository (e.g. <code class="docutils literal notranslate"><span class="pre">protocols</span></code>)</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/startup_checklist.html#existing-data-driven-repository-e-g-tangles-model">Existing, data-driven repository (e.g. <code class="docutils literal notranslate"><span class="pre">tangles_model</span></code>)</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="../../bootcamp/iap/the_shell.html">On Terminals and Shells</a><ul>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/the_shell.html#motivation">Motivation</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/the_shell.html#the-basics">The basics</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/the_shell.html#interface-basics">Interface basics</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/the_shell.html#starting-off-at-home">Starting off at home</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/the_shell.html#moving-away-from-home">Moving away from home</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/the_shell.html#relative-and-absolute-paths">Relative and absolute paths</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/the_shell.html#file-operations">File operations</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/the_shell.html#editing-and-viewing-files">Editing and viewing files</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/the_shell.html#finding-things-in-documents">Finding things in documents</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/the_shell.html#exercise">Exercise</a></li>
<li class="toctree-l4"><a class="reference internal" href="../../bootcamp/iap/the_shell.html#extras">Extras</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../bootcamp/ode_training/index.html">Gene expression modeling with ODEs</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../bootcamp/ode_training/ode_tutorial1.html">Tutorial 1: Modeling gene expression with differential equations</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../bootcamp/ode_training/ode_tutorial2.html">Tutorial 2: Examining parameters affecting our models</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../bootcamp/ode_training/ode_tutorial3.html">Tutorial 3: Michaelis-Menten Kinetics</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../bootcamp/ode_training/ode_tutorial4.html">Tutorial 4: Modeling repressors</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../bootcamp/ode_training/ode_tutorial5.html">Tutorial 5: Hill functions</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../bootcamp/training/index.html">“How to” Training Series</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../bootcamp/training/tutorial1.html">Tutorial 1: How to Write a Proposal</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../bootcamp/training/tutorial2.html">Tutorial 2: How to Read a Paper</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../bootcamp/training/tutorial3.html">Tutorial 3: How to Review a Paper</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../bootcamp/training/tutorial4.html">Tutorial 4: Graphic Figure Design</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../bootcamp/training/tutorial5.html">Tutorial 5: How to Compose Figures</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../../tech/index.html">Tech documentation</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../../tech/repo_setup.html">Repository setup</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../../contributor_guide.html">Contributor guide</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../../contributor_guide.html#local-environment-setup">Local environment setup</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#text-editor">Text editor</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#git">git</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#python-sphinx-setup">Python/Sphinx setup</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../contributor_guide.html#standard-workflow">Standard workflow</a></li>
<li class="toctree-l2"><a class="reference internal" href="../../contributor_guide.html#local-building">Local building</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#local-previewing">Local previewing</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#in-case-of-build-errors">In case of build errors</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../../contributor_guide.html#online-editing-through-github">Online editing through Github</a></li>
<li class="toctree-l2"><a class="reference internal" href="../../contributor_guide.html#repository-layout">Repository layout</a></li>
<li class="toctree-l2"><a class="reference internal" href="../../contributor_guide.html#basics-of-restructuredtext">Basics of reStructuredText</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#simple-markup">Simple markup</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#explicit-markup">Explicit markup</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#admonitions">Admonitions</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#code">Code</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#tables">Tables</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#references-and-links">References and links</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#images">Images</a></li>
<li class="toctree-l3"><a class="reference internal" href="../../contributor_guide.html#math">Math</a></li>
</ul>
</li>
</ul>
</li>
</ul>

            
          
        </div>
        
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../../index.html">Galloway Lab Protocols</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          

















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="../../index.html" class="icon icon-home"></a> &raquo;</li>
        
          <li><a href="../index.html">Protocols</a> &raquo;</li>
        
          <li><a href="index.html">Biochemical and analytical protocols</a> &raquo;</li>
        
      <li>CellBaum User’s Guide</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
          
            <a href="../../_sources/protocols/biochem_and_analytics/cellbaum.rst.txt" rel="nofollow"> View page source</a>
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="cellbaum-user-s-guide">
<h1>CellBaum User’s Guide<a class="headerlink" href="#cellbaum-user-s-guide" title="Permalink to this heading">¶</a></h1>
<p>A practical guide to CellBaum for those who want to live track cells but know absolutely nothing about computers.</p>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p>You can use CellBaum either on your computer or the MIT Supercloud.</p>
</div>
<div class="section" id="local-usage">
<h2>Local Usage<a class="headerlink" href="#local-usage" title="Permalink to this heading">¶</a></h2>
<div class="section" id="getting-started">
<h3>Getting started<a class="headerlink" href="#getting-started" title="Permalink to this heading">¶</a></h3>
<ol class="arabic">
<li><p>Follow Chris’s instructions for getting CellBaum set up on your computer and update this protocol with those instructions. (I don’t remember how to do this - I think you need to clone the CellBaum repository or something?)</p></li>
<li><p>Make sure you have CellProfiler and Fiji installed on your computer.</p></li>
<li><p>Download all the timelapse images to your computer so they can be accessed quickly. This can be done by right clicking on the folder and selecting “Always keep on this device”.
After you are done analyzing the images you can remove them from your computer by right clicking on the folder and selecting “Free up space”.</p></li>
<li><p>Complete the cellbaum_config.yml file for your specific experiment. This can be done using VS Code. See more detailed instructions below.</p></li>
<li><p>Open up a terminal in the CellBaum folder. You may open the terminal using VS Code.</p></li>
<li><p>Activate the virtual environment by typing conda activate ./cenv in the terminal:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">conda</span> <span class="n">activate</span> <span class="o">./</span><span class="n">cenv</span>
</pre></div>
</div>
</li>
<li><p>To do a dry-run the CellBaum pipline, type snakemake - -dry-run in the terminal. This is to double check that everything is configured correctly:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">snakemake</span> <span class="o">--</span><span class="n">dry</span><span class="o">-</span><span class="n">run</span>
</pre></div>
</div>
</li>
<li><p>To run CellBaum, type snakemake into the terminal. To assign a set number of cores on your computer, add the -j# flag after snakemake. For example, to use 7 cores on your computer to run CellBaum, type snakemake -j7:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">snakemake</span> <span class="o">-</span><span class="n">j7</span>
</pre></div>
</div>
</li>
<li><p>Let CellBaum run. This may take many hours. The typical order of jobs is:</p></li>
</ol>
<blockquote>
<div><ol class="arabic simple">
<li><p>folder merging (if needed, e.g. you stopped and re-started the timelapse)</p></li>
<li><p>grayscaling</p></li>
<li><p>find focus (if z-stacking was used)</p></li>
<li><p>stitching (if stitching was used)</p></li>
<li><p>run Cell Profiler pipeline to collect cell data and locations (nuclei_masking.cppipe)</p></li>
<li><p>btracking (for more information on Bayesian Tracking see <a class="reference external" href="https://github.com/quantumjot/BayesianTracker">GitHub</a> and their <a class="reference external" href="https://btrack.readthedocs.io/en/latest/">User’s Guide</a>)</p></li>
<li><p>Add cell data to HDF tracks file</p></li>
</ol>
</div></blockquote>
<div class="admonition tip">
<p class="admonition-title">Tip</p>
<p>If you want to stop CellBaum from running, hit ctrl+C while in the terminal and CellBaum will stop running after it finishes all the jobs it is currently running.</p>
<p>If you are too impatient to wait for CellBaum to finish running or it is stuck on the btracking step, you can kill the terminal.
However, if you kill the terminal, you will need to unlock it before you can run again:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">snakemake</span> <span class="o">--</span><span class="n">unlock</span>
</pre></div>
</div>
<p>You may also need to use the rerun-incomplete flag after unlocking if you stopped CellBaum partway through a job such as grayscaling:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">snakemake</span> <span class="o">-</span><span class="n">j7</span> <span class="o">--</span><span class="n">rerun</span><span class="o">-</span><span class="n">incomplete</span>
</pre></div>
</div>
</div>
</div>
<div class="section" id="cellbaum-config-file">
<h3>CellBaum Config File<a class="headerlink" href="#cellbaum-config-file" title="Permalink to this heading">¶</a></h3>
<p>If you have not completed a config file previously, copy the cellbaum_config.yml.template file and remove “.template” from the file name. This will be your config file.
If you have a previous config file, you may simply edit that one.</p>
<p>Complete the config file with the following information:</p>
<ul>
<li><p>directories</p>
<ul>
<li><p>Put the path on your computer to each item.</p></li>
<li><p>data_dir = Keyence data folder</p></li>
<li><p>cp_dir = CellProfiler app</p></li>
<li><p>fiji_dir = Fiji app</p></li>
<li><p>pipe_dir = pipeline for CellBaum (in CellBaum repository)</p></li>
<li><p>output_dir = where you want to save all output files</p></li>
<li><p>log_dir = where to save logs, usually this can be in an output_dir/Logs folder</p></li>
<li><p>cell_config = btrack config file, under the models folder in CellBaum repository</p></li>
<li><p>See examples below:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">data_dir</span><span class="p">:</span> <span class="n">C</span><span class="p">:</span>\<span class="n">Users</span>\<span class="n">ChemeGrad2020</span>\<span class="n">Massachusetts</span> <span class="n">Institute</span> <span class="n">of</span> <span class="n">Technology</span>\<span class="n">GallowayLab</span><span class="o">-</span><span class="n">Timelapse</span> <span class="o">-</span> <span class="n">Brittany</span>\<span class="mf">2022.12</span><span class="o">.</span><span class="mi">16</span><span class="n">_Hb9</span><span class="o">-</span><span class="n">Activation</span>\
<span class="n">cp_dir</span><span class="p">:</span> <span class="n">C</span><span class="p">:</span>\<span class="n">Program</span> <span class="n">Files</span>\<span class="n">CellProfiler</span>
<span class="n">fiji_dir</span><span class="p">:</span> <span class="n">C</span><span class="p">:</span>\<span class="n">Users</span>\<span class="n">ChemeGrad2020</span>\<span class="n">Documents</span>\<span class="n">Fiji</span><span class="o">.</span><span class="n">app</span>
<span class="n">pipe_dir</span><span class="p">:</span> <span class="n">C</span><span class="p">:</span>\<span class="n">Users</span>\<span class="n">ChemeGrad2020</span>\<span class="n">Documents</span>\<span class="n">GitHub</span>\<span class="n">CellBaum</span>\<span class="n">cp_files</span>
<span class="n">output_dir</span><span class="p">:</span> <span class="n">C</span><span class="p">:</span>\<span class="n">Users</span>\<span class="n">ChemeGrad2020</span>\<span class="n">Documents</span>\<span class="n">CellBaum_Analysis</span>\<span class="mf">2022.12</span><span class="o">.</span><span class="mi">16</span><span class="n">_Hb9</span><span class="o">-</span><span class="n">Activation</span>
<span class="n">log_dir</span><span class="p">:</span> <span class="n">C</span><span class="p">:</span>\<span class="n">Users</span>\<span class="n">ChemeGrad2020</span>\<span class="n">Documents</span>\<span class="n">CellBaum_Analysis</span>\<span class="mf">2022.12</span><span class="o">.</span><span class="mi">16</span><span class="n">_Hb9</span><span class="o">-</span><span class="n">Activation</span>\<span class="n">Logs</span>
<span class="n">cell_config</span><span class="p">:</span> <span class="n">C</span><span class="p">:</span>\<span class="n">Users</span>\<span class="n">ChemeGrad2020</span>\<span class="n">Documents</span>\<span class="n">GitHub</span>\<span class="n">CellBaum</span>\<span class="n">models</span>\<span class="n">cell_config</span><span class="o">.</span><span class="n">json</span>
</pre></div>
</div>
</li>
</ul>
</li>
<li><p>folder_merging_needed</p>
<ul class="simple">
<li><p>false if all your images are in a single folder from the Keyence</p></li>
<li><p>true if you need to merge two or more folders, for example you re-started the timelapse on the Keyence.</p></li>
<li><p>If true, you must include list under folders_to_merge</p></li>
</ul>
</li>
<li><p>gray_channels</p>
<ul>
<li><p>list of channels you want to grayscale and which grayscale method to use.</p></li>
<li><p>‘max’ uses the RGB channel in which the maximum pixel value occurs (I think)</p></li>
<li><p>‘avg’ uses the average of all three RGB channels (I think)</p></li>
<li><p>using a list of 3 ratios lets you choose the weight of each RGB channel (I think)</p></li>
<li><p>For example:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">gray_channels</span><span class="p">:</span>
    <span class="n">CH2</span><span class="p">:</span> <span class="nb">max</span>
    <span class="n">CH3</span><span class="p">:</span> <span class="nb">max</span>
    <span class="n">CH4</span><span class="p">:</span> <span class="n">avg</span>
    <span class="n">Overlay</span><span class="p">:</span> <span class="n">avg</span>
</pre></div>
</div>
</li>
</ul>
</li>
<li><p>focus_finding_needed</p>
<ul class="simple">
<li><p>True if you used z-stacking during your timelapse.</p></li>
<li><p>If true, you also need to include focus_channels.</p></li>
</ul>
</li>
<li><p>image_regex</p>
<ul>
<li><p>A regular expression that extracts data from image names such as time, well, stitching position, z-stack, and channel.
You can check if your regular expression fits your file names using <a class="reference external" href="https://regex101.com/">https://regex101.com/</a>. For example, your regular expression could be:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span>image_regex: (?P&lt;prefix&gt;.*)_(?P&lt;time&gt;T\d{4})_(?P&lt;well&gt;XY\d{2})_(?P&lt;position&gt;\d{5})_Z(?P&lt;stack&gt;\d{3})_(?P&lt;channel&gt;.*)\.tif
</pre></div>
</div>
</li>
</ul>
</li>
<li><p>focus_channels</p>
<ul>
<li><p>List of channels for which you want to find the most in-focus z-slice. For example:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">focus_channels</span><span class="p">:</span>
 <span class="o">-</span> <span class="n">CH2</span>
 <span class="o">-</span> <span class="n">CH3</span>
 <span class="o">-</span> <span class="n">CH4</span>
 <span class="o">-</span> <span class="n">Overlay</span>
</pre></div>
</div>
</li>
</ul>
</li>
<li><p>example_image_name</p>
<ul>
<li><p>An example name of your image to check your regular expression above. You can just copy paste a file name of an image from your data folder:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">example_image_name</span><span class="p">:</span> <span class="mi">10</span><span class="n">x_T0001_XY01_00001_Z001_CH2</span><span class="o">.</span><span class="n">tif</span>
</pre></div>
</div>
</li>
</ul>
</li>
<li><p>pre_stitch_correction_needed</p>
<ul class="simple">
<li><p>True if you want to make some kind of correction to your images before stitching.
For example you could apply a threshold filter to the image and reduce noise.</p></li>
<li><p>If doing pre-stitch corrections, you will need to update the img_processing.cppipe in CellProfiler to make the desired pre-stitch corrections to your images.</p></li>
</ul>
</li>
<li><p>stitching / prefix / template</p>
<ul>
<li><p>Needs to be included if you used stitching during your timelapse.</p></li>
<li><p>grid_width and grid_height are how many images are in each dimension of your stitching grid.</p></li>
<li><p>Prefix is a list of which channels you want to stitch.</p></li>
<li><p>Template is which channel in the list you want to use as a template (zero indexed!). It is probably best to use a channel with the most “information” as a template.</p></li>
<li><p>Example usage:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">stitching</span><span class="p">:</span>
    <span class="n">grid_width</span><span class="p">:</span> <span class="mi">3</span>
    <span class="n">grid_height</span><span class="p">:</span> <span class="mi">4</span>
    <span class="n">Prefix</span><span class="p">:</span>
    <span class="o">-</span> <span class="n">CH2</span>
    <span class="o">-</span> <span class="n">CH3</span>
    <span class="o">-</span> <span class="n">CH4</span>
    <span class="o">-</span> <span class="n">Overlay</span>
    <span class="n">Template</span><span class="p">:</span> <span class="mi">3</span>
</pre></div>
</div>
</li>
</ul>
</li>
<li><p>minsize/maxsize</p>
<ul>
<li><p>The expected minimum and maximum size of objects/nuclei being tracked (pixel diameter). For example:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">minsize</span><span class="p">:</span> <span class="mi">7</span>
<span class="n">maxsize</span><span class="p">:</span> <span class="mi">45</span>
</pre></div>
</div>
</li>
</ul>
</li>
<li><p>Update_method</p>
<ul>
<li><p>Which method to use when tracking objects (EXACT or APPROXIMATE).
It is recommended to start with the default EXACT. This option considers all possible combinations of linking objects, so it can be slow for very large datasets.
If there are issues converging to switch to APPROXIMATE.</p></li>
<li><p>In general, for cell datasets with &lt;1000 cells per time point we recommend EXACT. If you have larger datasets, we recommend APPROXIMATE.</p></li>
<li><p>Example usage:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">Update_method</span><span class="p">:</span> <span class="n">APPROXIMATE</span>
</pre></div>
</div>
</li>
</ul>
</li>
<li><p>Max_search_radius</p>
<ul>
<li><p>The maximum search radius for the tracking algorithm in isotropic unit of the data. This parameter can be used to prevent very large displacements when linking objects.</p></li>
<li><p>Example usage:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">Max_search_radius</span><span class="p">:</span> <span class="mi">100</span>
</pre></div>
</div>
</li>
</ul>
</li>
<li><p>Volume</p>
<ul>
<li><p>An estimate of the dimensions of the imaging volume, used to define the edges of the field of view for generating hypotheses and labeling tracks as lost.</p></li>
<li><p>This is pretty much the dimensions of the stitched images in pixels.</p></li>
<li><p>It is recommend to use ‘auto’ which automatically calculates these values based on the size of the images.</p></li>
<li><p>Example usage:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">Volume</span><span class="p">:</span> <span class="n">auto</span>
</pre></div>
</div>
</li>
<li><p>OR:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">Volume</span><span class="p">:</span>
    <span class="n">x</span><span class="p">:</span>
    <span class="o">-</span> <span class="mi">0</span>
    <span class="o">-</span> <span class="mi">4577</span>
    <span class="n">y</span><span class="p">:</span>
    <span class="o">-</span> <span class="mi">0</span>
    <span class="o">-</span> <span class="mi">5427</span>
    <span class="n">z</span><span class="p">:</span>
    <span class="o">-</span> <span class="mi">0</span>
    <span class="o">-</span> <span class="mi">2</span>
</pre></div>
</div>
</li>
</ul>
</li>
<li><p>Step_size</p>
<ul>
<li><p>Not really sure what this does. It doesn’t seem to affect anything. You can just put like 100 or something:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">Step_size</span><span class="p">:</span> <span class="mi">100</span>
</pre></div>
</div>
</li>
</ul>
</li>
<li><p>CP_Data_Keep</p>
<ul>
<li><p>List of which CellProfiler data to keep, or you can just put ‘all’ for simplicity:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">CP_Data_Keep</span><span class="p">:</span> <span class="s1">&#39;all&#39;</span>
</pre></div>
</div>
</li>
</ul>
</li>
<li><p>time_interval_to_track</p>
<ul>
<li><p>This feature is useful if the btracker isn’t converging you want to change the parameters in the btracker config file (models/cell_config.json).</p></li>
<li><p>Choose which time points to track. For example, if you have 181 time points and you want to see if you can get it to converge on the subset of time points from 0 to 50 you could put:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">time_interval_to_track</span><span class="p">:</span>
    <span class="nb">min</span><span class="p">:</span> <span class="mi">0</span>
    <span class="nb">max</span><span class="p">:</span> <span class="mi">50</span>
</pre></div>
</div>
</li>
</ul>
</li>
</ul>
</div>
<div class="section" id="getting-the-btracker-to-converge">
<h3>Getting the btracker to Converge<a class="headerlink" href="#getting-the-btracker-to-converge" title="Permalink to this heading">¶</a></h3>
<p>The best way to make sure CellBaum will converge is by carefully choosing the conditions of your timelapse. Two recommendations are:</p>
<ol class="arabic simple">
<li><p>Adding a population of dark cells and/or seeding cells sparsely will help with tracking. Many tracking issues are due to simply way too many cells in the field of view.</p></li>
<li><p>Imaging more frequently can help when cells move around a lot. However, you may start worrying about photo bleaching or computer storage space restrictions if you image too frequently.</p></li>
</ol>
<p>Ensure your nuclei masking pipeline works well with your stitched images.</p>
<ul class="simple">
<li><p>It is required that you always re-do the CellProfiler pipeline nuclei_masking.cppipe for your specific images AFTER they have been stitched.</p></li>
<li><p>You will probably need to stop CellBaum partway through in order to update the pipeline and use stitched images.</p></li>
<li><p>Stitching will affect the RegEx used in your CellProfiler pipeline.</p></li>
<li><p>You may use different channels between experiments, nuclei will be different size ranges, there may be weird background luminance, etc.</p></li>
</ul>
<p>Change the btrack model parameters (models/cell_config.json)</p>
<ul class="simple">
<li><p>You can read more about the btrack config models <a class="reference external" href="https://btrack.readthedocs.io/en/latest/user_guide/configuration.html#">here</a>.</p></li>
<li><p>The output of the tracking is very sensitive to the choice of parameter values. The global optimization step can take a very long time to complete if you have a poor choice of model parameters.</p></li>
<li><p>Changing the Motion Model probably won’t help much.</p></li>
<li><p>Brittany has found that increasing the values for lambda_link and lambda_branch helped most with converging.</p></li>
</ul>
</div>
</div>
<div class="section" id="mit-supercloud-usage">
<h2>MIT Supercloud Usage<a class="headerlink" href="#mit-supercloud-usage" title="Permalink to this heading">¶</a></h2>
<div class="section" id="getting-access-to-the-mit-supercloud">
<h3>Getting Access to the MIT Supercloud<a class="headerlink" href="#getting-access-to-the-mit-supercloud" title="Permalink to this heading">¶</a></h3>
<ol class="arabic simple">
<li><p>Follow the directions <a class="reference external" href="https://supercloud.mit.edu/requesting-account">on the MIT Supercloud website</a> for getting access to the MIT supercloud.</p></li>
<li><p>First you will need to fill out the Account Request Form.</p></li>
<li><p>Then you will need to send an email to <a class="reference external" href="mailto:supercloud&#37;&#52;&#48;mit&#46;edu">supercloud<span>&#64;</span>mit<span>&#46;</span>edu</a> and CC Katie.</p></li>
<li><p>Then you might have to do some other setup stuff. Ask Chris for help and then add those instructions to this protocol.</p></li>
</ol>
<div class="admonition hint">
<p class="admonition-title">Hint</p>
<p>You may use this as a template for your email:</p>
<p>Hello,</p>
<p>I have submitted an account request form so I can get access to the MIT Supercloud for my research project. I will be using the Supercloud to analyze and generate data from time-lapse microscopy images. I have Cc’d my PI, Katie Galloway, so she can confirm that I will need access for my work.</p>
<p>Thanks,</p>
<p>Tony Stark</p>
</div>
</div>
<div class="section" id="upload-timelapse-files-from-the-keyence-to-the-supercloud">
<h3>Upload Timelapse Files from the Keyence to the Supercloud<a class="headerlink" href="#upload-timelapse-files-from-the-keyence-to-the-supercloud" title="Permalink to this heading">¶</a></h3>
<ol class="arabic">
<li><p>Open a terminal <strong>in your Keyence Timelapse Ondrive folder</strong> (shift + right click then choose Open Terminal).</p></li>
<li><p>Use rclone to copy your experiment folder to the supercloud. Rclone uses the following command to copy files:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">rclone</span> <span class="n">copy</span> <span class="p">(</span><span class="o">--</span><span class="n">dry</span><span class="o">-</span><span class="n">run</span><span class="p">)</span> <span class="p">(</span><span class="o">--</span><span class="n">progress</span><span class="p">)</span> <span class="n">source</span><span class="p">:</span><span class="n">sourcepath</span> <span class="n">dest</span><span class="p">:</span><span class="n">destpath</span>
</pre></div>
</div>
</li>
<li><p>It is recommended to first do a dry run to ensure everything will transfer correctly:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="o">~/</span><span class="n">downloads</span><span class="o">/</span><span class="n">rclone</span><span class="o">-</span><span class="n">v1</span><span class="o">.</span><span class="mf">60.1</span><span class="o">-</span><span class="n">windows</span><span class="o">-</span><span class="n">amd64</span><span class="o">/</span><span class="n">rclone</span><span class="o">-</span><span class="n">v1</span><span class="o">.</span><span class="mf">60.1</span><span class="o">-</span><span class="n">windows</span><span class="o">-</span><span class="n">amd64</span><span class="o">/</span><span class="n">rclone</span><span class="o">.</span><span class="n">exe</span> <span class="n">copy</span> <span class="o">--</span><span class="n">dry</span><span class="o">-</span><span class="n">run</span> <span class="mf">2022.12</span><span class="o">.</span><span class="mi">16</span><span class="n">_Hb9</span><span class="o">-</span><span class="n">Activation</span> <span class="n">supercloud</span><span class="o">-</span><span class="n">blende</span><span class="p">:</span><span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">gridsan</span><span class="o">/</span><span class="n">blende</span><span class="o">/</span><span class="n">galloway_shared</span><span class="o">/</span><span class="n">data</span><span class="o">/</span><span class="mf">2022.12</span><span class="o">.</span><span class="mi">16</span><span class="n">_Hb9</span><span class="o">-</span><span class="n">Activation</span>
</pre></div>
</div>
</li>
<li><p>Then when you are ready to transfer the files, include the progress flag to see upload progress:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="o">~/</span><span class="n">downloads</span><span class="o">/</span><span class="n">rclone</span><span class="o">-</span><span class="n">v1</span><span class="o">.</span><span class="mf">60.1</span><span class="o">-</span><span class="n">windows</span><span class="o">-</span><span class="n">amd64</span><span class="o">/</span><span class="n">rclone</span><span class="o">-</span><span class="n">v1</span><span class="o">.</span><span class="mf">60.1</span><span class="o">-</span><span class="n">windows</span><span class="o">-</span><span class="n">amd64</span><span class="o">/</span><span class="n">rclone</span><span class="o">.</span><span class="n">exe</span> <span class="n">copy</span> <span class="o">--</span><span class="n">progress</span> <span class="mf">2022.12</span><span class="o">.</span><span class="mi">16</span><span class="n">_Hb9</span><span class="o">-</span><span class="n">Activation</span> <span class="n">supercloud</span><span class="o">-</span><span class="n">blende</span><span class="p">:</span><span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">gridsan</span><span class="o">/</span><span class="n">blende</span><span class="o">/</span><span class="n">galloway_shared</span><span class="o">/</span><span class="n">data</span><span class="o">/</span><span class="mf">2022.12</span><span class="o">.</span><span class="mi">16</span><span class="n">_Hb9</span><span class="o">-</span><span class="n">Activation</span>
</pre></div>
</div>
</li>
</ol>
<div class="admonition note">
<p class="admonition-title">Note</p>
<p>rclone is safe to interrupt at any point.</p>
<p>On the Keyence computer, rclone isn’t in the Path, so you need to do the whole path for rclone.exe in the Downloads folder.</p>
<p>Remember to include add supercloud-blende: (or your equivalent) before the dest path to specify you are using a remote path.</p>
<p>It can take several hours for all the files to transfer to the supercloud.</p>
</div>
</div>
<div class="section" id="using-the-mit-supercloud">
<h3>Using the MIT Supercloud<a class="headerlink" href="#using-the-mit-supercloud" title="Permalink to this heading">¶</a></h3>
<ol class="arabic">
<li><p>Open powershell.</p></li>
<li><p>In the terminal type:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">ssh</span> <span class="n">supercloud</span>
</pre></div>
</div>
</li>
<li><p>You may need to type your password. If you don’t want to type your password every time you can use ssh-add to use an agent to control authentification.</p></li>
<li><p>Navigate to the CellBaum folder:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">cd</span> <span class="o">~/</span><span class="n">galloway_shared</span><span class="o">/</span><span class="n">CellBaum</span>
</pre></div>
</div>
</li>
<li><p>Do a git pull to update CellBaum:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">git</span> <span class="n">pull</span>
</pre></div>
</div>
</li>
<li><p>If you have your own branch in CellBaum, switch to your branch. For example:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">git</span> <span class="n">checkout</span> <span class="n">KTRs</span>
</pre></div>
</div>
</li>
<li><p>Update the supercloud_config.yml file for your specific timelapse. You can update the file in VS Code on your computer, push it to GitHub, and then do git pull on the supercloud. Or you can use nano:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">nano</span> <span class="n">supercloud_config</span><span class="o">.</span><span class="n">yml</span>
</pre></div>
</div>
</li>
<li><p>You might also need to update the snakefile to use the supercloud_config.yml instead of the cellbaum_config.yml file:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">nano</span> <span class="n">snakefile</span>

<span class="c1"># Then change</span>

<span class="n">configfile</span><span class="p">:</span> <span class="s2">&quot;cellbaum_config.yml&quot;</span>

<span class="c1"># to</span>

<span class="n">configfile</span><span class="p">:</span> <span class="s2">&quot;supercloud_config.yml&quot;</span>
</pre></div>
</div>
</li>
<li><p>Update any CellProfiler pipelines that are used. You can copy files to the supercloud via <a class="reference external" href="https://supercloud.mit.edu/accessing-and-transferring-data-and-files">scp</a>, or you can git push the file from your computer and then git pull from the supercloud.</p></li>
<li><p>To run CellBaum you need to run the submission bash script:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">LLsub</span> <span class="n">sub_script</span><span class="o">.</span><span class="n">sh</span>
</pre></div>
</div>
</li>
<li><p>To see what is running, you can type:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">LLstat</span>
</pre></div>
</div>
</li>
<li><p>When you want to leave the supercloud, just type exit:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">exit</span>
</pre></div>
</div>
</li>
<li><p>You can view the files on the supercloud using the Powershell and ls. Or you can log into the online web portal: <a class="reference external" href="https://txe1-portal.mit.edu/">https://txe1-portal.mit.edu/</a></p></li>
</ol>
</div>
<div class="section" id="supercloud-config">
<h3>Supercloud Config<a class="headerlink" href="#supercloud-config" title="Permalink to this heading">¶</a></h3>
<p>This config file is virtually the same as the cellbaum_config.yml file, except you will want to change all the directories to locations on the supercloud. For example:</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">data_dir</span><span class="p">:</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">gridsan</span><span class="o">/</span><span class="n">blende</span><span class="o">/</span><span class="n">galloway_shared</span><span class="o">/</span><span class="n">data</span><span class="o">/</span><span class="mf">2022.12</span><span class="o">.</span><span class="mi">16</span><span class="n">_Hb9</span><span class="o">-</span><span class="n">Activation</span><span class="o">/</span><span class="mf">2022.12</span><span class="o">.</span><span class="mi">16</span><span class="n">_Hb9</span><span class="o">-</span><span class="n">Activation</span>
<span class="n">cp_dir</span><span class="p">:</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">gridsan</span><span class="o">/</span><span class="n">blende</span><span class="o">/</span><span class="n">galloway_shared</span><span class="o">/</span><span class="n">cluster_infrastructure</span><span class="o">/</span><span class="n">conda_envs</span><span class="o">/</span><span class="n">cellbaum</span><span class="o">/</span><span class="nb">bin</span><span class="o">/</span>
<span class="n">fiji_dir</span><span class="p">:</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">gridsan</span><span class="o">/</span><span class="n">blende</span><span class="o">/</span><span class="n">galloway_shared</span><span class="o">/</span><span class="nb">bin</span><span class="o">/</span><span class="n">Fiji</span><span class="o">.</span><span class="n">app</span>
<span class="n">pipe_dir</span><span class="p">:</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">gridsan</span><span class="o">/</span><span class="n">blende</span><span class="o">/</span><span class="n">galloway_shared</span><span class="o">/</span><span class="n">CellBaum</span><span class="o">/</span><span class="n">cp_files</span>
<span class="n">output_dir</span><span class="p">:</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">gridsan</span><span class="o">/</span><span class="n">blende</span><span class="o">/</span><span class="n">galloway_shared</span><span class="o">/</span><span class="n">data</span><span class="o">/</span><span class="mf">2022.12</span><span class="o">.</span><span class="mi">16</span><span class="n">_Hb9</span><span class="o">-</span><span class="n">Activation</span><span class="o">/</span><span class="n">CellBaum_results</span>
<span class="n">log_dir</span><span class="p">:</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">gridsan</span><span class="o">/</span><span class="n">blende</span><span class="o">/</span><span class="n">galloway_shared</span><span class="o">/</span><span class="n">data</span><span class="o">/</span><span class="mf">2022.12</span><span class="o">.</span><span class="mi">16</span><span class="n">_Hb9</span><span class="o">-</span><span class="n">Activation</span><span class="o">/</span><span class="n">CellBaum_results</span><span class="o">/</span><span class="n">Logs</span>
<span class="n">cell_config</span><span class="p">:</span> <span class="o">/</span><span class="n">home</span><span class="o">/</span><span class="n">gridsan</span><span class="o">/</span><span class="n">blende</span><span class="o">/</span><span class="n">galloway_shared</span><span class="o">/</span><span class="n">CellBaum</span><span class="o">/</span><span class="n">models</span><span class="o">/</span><span class="n">cell_config</span><span class="o">.</span><span class="n">json</span>
</pre></div>
</div>
</div>
</div>
<div class="section" id="data-analysis">
<h2>Data Analysis<a class="headerlink" href="#data-analysis" title="Permalink to this heading">¶</a></h2>
<p>To add later</p>
<p>How to open HDF files and understand LBEPR format.</p>
</div>
</div>


           </div>
           
          </div>
          <footer>

  <hr/>

  <div role="contentinfo">
    <p>
        &#169; Copyright 2024, Galloway Lab. Shared under the Creative Commons 4.0 Attribution International license.
      <span class="lastupdated">
        Last updated on Jun 08, 2023.
      </span>

    </p>
  </div>
    
    
    
    Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    
    provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>
        </div>
      </div>

    </section>

  </div>
  

  <div class="rst-versions" data-toggle="rst-versions" role="note" aria-label="versions">
    <span class="rst-current-version" data-toggle="rst-current-version">
      <span class="fa fa-book"> Downloads and versions</span>
      v: ehs_safety
      <span class="fa fa-caret-down"></span>
    </span>
    <div class="rst-other-versions">
      
      
      <dl>
        <dt>Versions</dt>
        
           <strong> 
          <dd><a href="/protocols/en/ehs_safety/">ehs_safety</a></dd>
           </strong> 
        
          
          <dd><a href="/protocols/en/latest/">latest</a></dd>
          
        
          
          <dd><a href="/protocols/en//"></a></dd>
          
        
      </dl>
      
      
      <dl>
        <dt>Downloads</dt>
        
          <dd><a href="/protocols/en/ehs_safety/galloway_lab_protocols.pdf">pdf</a></dd>
        
      </dl>
      
      
      <hr/>
      Hosted with <span class="fa fa-heart"></span> on GitHub Pages.

    </div>
  </div>


  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>

  
  
    
   

</body>
</html>